test/utils/contracts.js

```diff
@@ -8,4 +8,10 @@ module.exports = {
             lenderData.lastAccruedInterest
         );
     },
+    createInstance: async newPromise => {
+        const contract = await newPromise;
+        assert(contract);
```

Comment:
For what reason is `assert(contract)` and `assert(contract.address)` needed? If the contract fails to deploy, then `await new ContractName()` fails and throws an error. What do these lines catch?